ui/components/ui/identicon/identicon.component.js

```diff
@@ -76,6 +76,10 @@ export default class Identicon extends PureComponent {
     const { className, diameter, alt, imageBorder, ipfsGateway } = this.props;
     let { image } = this.props;
 
+    if (Array.isArray(image) && image.length) {
+      image = image[0];
+    }
+
     if (image.toLowerCase().startsWith('ipfs://')) {
```

Comment:
maybe just in case let's do a type check here too? 
```suggestion
    if (typeof image === 'string' && image.toLowerCase().startsWith('ipfs://')) {
```

Reply:
sgtm, done